app/src/main/java/com/android/josesantos/upcomingmovies/data/api/upcommingmovies/MoviesDbRoutes.java

```diff
@@ -0,0 +1,34 @@
+package com.android.josesantos.upcomingmovies.data.api.upcommingmovies;
+
+import com.android.josesantos.upcomingmovies.data.entities.Genres;
+import com.android.josesantos.upcomingmovies.data.entities.MovieConfiguration;
+import com.android.josesantos.upcomingmovies.data.entities.PageResponse;
+import com.android.josesantos.upcomingmovies.data.entities.Movie;
+
+import io.reactivex.Observable;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+/**
+ * Created by josesantos on 03/12/17.
+ */
+
+public interface MoviesDbRoutes {
+
+    @GET("discover/movie")
```

Comment:
We expected you to call the endpoint "movie/upcoming" . Even though the object returned is the same, the content is different